docs/src/increase-observability/integrate-observability/blackfire.md

```diff
@@ -29,11 +29,14 @@ All customers can also subscribe to Blackfire separately.
 
 ## Getting started
 
-### On a Grid plan or {{% names/dedicated-gen-3 %}} infrastructure
+### On a Grid plan or {{% names/dedicated-gen-3 %}} infrastructure 
 
 To enable Blackfire on your PHP or Python applications,
 follow [the step-by-step instructions](https://blackfire.io/docs/integrations/paas/platformsh).
 
+If you have subscribed to the Platform.sh [Observability Suite](https://platform.sh/features/observability-suite/), 
```

Comment:
:hammer_and_wrench: I think this works with fewer words, and I know you love fewer words.
```suggestion
If you have the [Observability Suite](https://platform.sh/features/observability-suite/), 
```

Reply:
I might consider moving this paragraph first. Otherwise I worry people go through the step-by-step instructions and only after realize they don't have to. Not sure if that's a reasonable fear, though.

Reply:
💯